src/Features/Core/Portable/InheritanceMargin/AbstractInheritanceMarginService_Helpers.cs

```diff
@@ -689,16 +605,15 @@ private static ImmutableArray<ISymbol> GetOverriddenSymbols(ISymbol memberSymbol
         /// Otherwise, create the full non-classified DefinitionItem. Because in such case we want to display all the locations to the user
         /// by reusing the FAR window.
         /// </summary>
-        private static DefinitionItem ToSlimDefinitionItem(ISymbol symbol, Solution solution)
+        private static DefinitionItem? ToSlimDefinitionItem(ISymbol symbol, Solution solution)
         {
-            RoslynDebug.Assert(IsNavigableSymbol(symbol));
             var locations = symbol.Locations;
             if (locations.Length > 1)
             {
                 return symbol.ToNonClassifiedDefinitionItem(
                     solution,
                     FindReferencesSearchOptions.Default with { UnidirectionalHierarchyCascade = true },
-                    includeHiddenLocations: false);
+                    includeHiddenLocations: true);
```

Comment:
how does this work?  we may not be able to navigate to hidden locations (for example, legacy razor locations that are hiddne).  so what happens if the user clicks on one of those items?

Reply:
I was thinking it only covers this case
```
#line hidden
// Some code
```
And I feel it not a problem to support that

Reply:
But if it includes things we don't want to support like legacy razor locations then I am fine to set this back to false. (since all these are edges cases and we have never heard any feedback around this)

Reply:
Change this back to false